ignition/boot/configurators/jobqueue.py

```diff
@@ -26,7 +27,12 @@ def configure(self, configuration, service_register):
                 raise ValueError('messaging.topics.job_queue must be set when bootstrap.job_queue.service_enabled is True')
             if messaging_config.topics.job_queue.name is None:
                 # Job Queue topic should be unique per VIM/Lifecycle driver cluster (not per instance) so we default the value at runtime to include the app name
-                messaging_config.topics.job_queue.name = '{0}_job_queue'.format(configuration.app_name.replace(" ", "_"))
+                safe_topic_name = re.sub('[^A-Za-z0-9-_ ]+', '', configuration.app_name)
+                # Remove any concurrent spaces
```

Comment:
How is this removing concurrent spaces (won't the regex do this)?

Reply:
The regex keeps spaces (there's a space character at the end of it, after the underscore) - then, as we've removed certain characters we can end up with some cases where there are multiple spaces, so the split separates the string by spaces, then re-joins them with a single space.